rosdep/base.yaml

```
@@ -6990,6 +6990,7 @@ spirv-headers:
   debian: [spirv-headers]
   fedora: [spirv-headers-devel]
   gentoo: [dev-util/spirv-headers]
+  rhel: [spirv-headers]
```

The source package is called `spirv-headers`, but the binary package being built from it is called `spirv-headers-devel`.
```suggestion
  rhel: [spirv-headers-devel]
```